src/main/java/io/cdap/plugin/gcp/firestore/source/util/FilterInfoParser.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.firestore.source.util;
+
+import com.google.common.base.Splitter;
+import io.cdap.plugin.gcp.firestore.util.Util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * {@link FilterInfoParser}
+ */
+public class FilterInfoParser {
+
+  /**
+   * @return the filters to apply. Returns an empty list if filters contains a macro. Otherwise, the list
```

Comment:
better explain what the filterString looks like

Reply:
Done